nav2_msgs/srv/GetCost.srv

```diff
@@ -0,0 +1,6 @@
+# Get costmap cost at given point
+
+float32 x
+float32 y
+---
+uint8 cost
```

Comment:
You cast the output to a float in the cpp code but its a uint?

Reply:
good point, I just copied it from [here](https://github.com/open-navigation/navigation2/blob/d0a65d5afbab3b758408b79ebf5cd344b3377b00/nav2_smac_planner/src/collision_checker.cpp#L114.) why is it cast there?

Reply:
Because footprint cost is a float.